pyproject.toml

```
@@ -42,9 +44,12 @@ flake8 = "^4.0.1"
 pytest-httpx = "^0.20.0"
 pytest-asyncio = "^0.18.2"
 async-asgi-testclient = "^1.4.10"
-pytest-cov = "^3.0.0"
 testcontainers = "3.4.2"
 pytest = "^7.1.2"
+pytest-cov = "^3.0.0"
+
+[tool.coverage.run]
+concurrency = ["greenlet"]
```

It also makes sense to add "thread" into the list. 
We have several places with `run_in_threadpool` that should be also considered in coverage report.

```suggestion
concurrency = ["thread", "greenlet"]
```

---

the coverage increased back to 82%. Thank you!